apps/backend/src/libs/modules/server-application/server-application.ts

```diff
@@ -2,6 +2,7 @@ import { config } from "~/libs/modules/config/config.js";
 import { database } from "~/libs/modules/database/database.js";
 import { logger } from "~/libs/modules/logger/logger.js";
 import { authController } from "~/modules/auth/auth.js";
+import { surveyController } from "~/modules/surveys/surveys.js";
```

Comment:
Let's name the module `onboarding` to have the consistency. `onboardingController`, `onboardingRepository`, etc.